test/kitchen/site-cookbooks/dd-security-agent-check/recipes/default.rb

```diff
@@ -0,0 +1,80 @@
+#
+# Cookbook Name:: dd-security-agent-check
+# Recipe:: default
+#
+# Copyright (C) 2020 Datadog
+#
+
+if node['platform_family'] != 'windows'
+  wrk_dir = '/tmp/security-agent'
+
+  directory wrk_dir do
+    recursive true
+  end
+
+  cookbook_file "#{wrk_dir}/testsuite" do
+    source "testsuite"
+    mode '755'
+  end
+
+  # To uncomment when gitlab runner are able to build with GOARCH=386
+  # cookbook_file "#{wrk_dir}/testsuite32" do
+  #   source "testsuite32"
+  #   mode '755'
+  # end
+
+  kernel_module 'loop' do
+    action :load
+  end
+
+  if not ['redhat', 'suse', 'opensuseleap'].include?(node[:platform])
+    docker_service 'default' do
+      action [:create, :start]
+    end
+
+    docker_image 'debian' do
+      tag 'bullseye'
+      action :pull
+    end
+
+    docker_container 'docker-testsuite' do
+      repo 'debian'
+      tag 'bullseye'
+      cap_add ['SYS_ADMIN', 'SYS_RESOURCE', 'SYS_PTRACE', 'NET_ADMIN', 'IPC_LOCK', 'ALL']
+      command "sleep 3600"
+      volumes '/tmp/security-agent:/tmp/security-agent'
+      privileged true
+    end
+
+    docker_exec 'debug_fs' do
+      container 'docker-testsuite'
+      command ['mount', '-t', 'debugfs', 'none', '/sys/kernel/debug']
+    end
+
+    for i in 0..7 do
+      docker_exec 'create_loop' do
+        container 'docker-testsuite'
+        command ['bash', '-c', "mknod /dev/loop#{i} b 7 #{i} || true"]
+      end
+    end
+  end
+
+  if not platform_family?('suse')
+    package 'Install i386 libc' do
+      case node[:platform]
+      when 'redhat', 'centos', 'fedora'
+        package_name 'glibc.i686'
+      when 'ubuntu', 'debian'
+        package_name 'libc6-i386'
+      # when 'suse'
```

Comment:
Is there a reason why this is commented?

Reply:
Yes, to not forget to do it when we'll be able to install packages on SUSE